recipes/aghast/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "aghast" %}
+{% set version = "0.1.0rc4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diana-hep/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: d458451970602df06722f388d6a2c4df2b83c149125fceea08bc3568e9ae3e1e
+  patches:
+    - 0001-Fix-packaging-for-conda.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install ./python --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - python-flatbuffers >=1.8.0
+    - numpy
+
+test:
+  source_files:
+    - python/tests
+  requires:
+    - pytest
+  imports:
+    - aghast
+  commands:
+    - py.test python/tests
+
+about:
+  home: https://github.com/diana-hep/aghast
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Aggregated, histogram-like statistics, shareable as Flatbuffers.'
+  description: |
+    Aghast is a histogramming library that does not fill histograms and does not plot them.
+    Its role is behind the scenes, to provide better communication between histogramming libraries.
+  doc_url: https://mybinder.org/v2/gh/diana-hep/aghast/master?urlpath=lab/tree/binder%2Fexamples.ipynb
+  dev_url: https://github.com/diana-hep/aghast
+
+extra:
+  recipe-maintainers:
+    - chrisburr
+    - jpivarski
```

Comment:
@jpivarski Can you confirm if you're happy to be listed as a maintainer?

I don't mind doing the maintenance like I do with awkward and uproot.

Reply:
Quite happy to be listed. (The only downside I've found is that the number of repos that I could possibly have in continuous integration taxes Travis and AppVeyor!) I want to be aware of how this works, at least.